src/utils/service_info_log.rs

```diff
@@ -0,0 +1,45 @@
+//! This module contains the function to log service information at startup.
+use log::info;
+use std::env;
+
+/// Logs service information at startup
+pub fn log_service_info() {
+    let service_name = env!("CARGO_PKG_NAME");
+    let service_version = env!("CARGO_PKG_VERSION");
+
+    info!("=== OpenZeppelin Relayer Service Starting ===");
+    info!("🚀 Service: {} v{}", service_name, service_version);
+    info!("🦀 Rust Version: {}", env!("CARGO_PKG_RUST_VERSION"));
```

Comment:
maybe the version of openzeppelin-relayer would be also useful here, especially for troubleshooting

Reply:
Thanks for feedback.

That is already included (service version)

```
 === OpenZeppelin Relayer Service Starting ===
13:15:28 [INFO] 🚀 Service: openzeppelin-relayer v1.0.0
13:15:28 [INFO] 🦀 Rust Version: 1.85
13:15:28 [INFO] 💻 Platform: macos
13:15:28 [INFO] 💻 Architecture: aarch64
13:15:28 [INFO] 📁 Working Directory: /Users/zeljkomarkovic/Documents/Projects/openzeppelin/openzeppelin-relayer
13:15:28 [INFO] 🔧 Log Level: DEBUG
13:15:28 [INFO] 🕒 Started at: 2025-07-14 13:15:28 UTC
13:15:28 [INFO] ℹ️ Visit the Relayer documentation for more information https://docs.openzeppelin.com/relayer/
```